src/platform/user/profile/vet360/components/base/TransactionErrorBanner.jsx

```diff
@@ -31,22 +26,14 @@ export function MVILookupFailError(props) {
     <AlertBox
       {...props}
       status="error"
-      content={
-        <div>
-          <h4>
-            We’re having trouble matching your information to our Veteran
-            records
-          </h4>
-          <p>
+      headline="We’re having trouble matching your information to our Veteran records"
+      content={`
```

Comment:
This content isn't going to render correctly. Try passing in some JSX to use as the content instead of just an interpolated string. Also, can you add the extra bottom margin on this `AlertBox`?

Reply:
I put the jsx back minus the headline